lib/simple_form/form_builder.rb

```diff
@@ -507,9 +507,10 @@ def find_input(attribute_name, options = {}, &block)
     # default alwayls fallback to the user :as option, or to a :select when a
     # collection is given.
     def default_input_type(attribute_name, column, options)
+      custom_type = find_custom_type(attribute_name.to_s) and return custom_type
+
       return options[:as].to_sym if options[:as]
```

Comment:
I think we should have it after this `as` condition, because that'd allow you to override custom type on a per-input basis, right?